blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarter.java

```diff
@@ -295,8 +296,10 @@ public void run() {
                     TimeUnit.MINUTES.toMillis(savePeriodMins));
         }
 
-        // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
-        new RestartGauge(Metrics.getRegistry(), RollupService.class);
+        if (!SkipInstantiateRestartGauge) {
+            // has the side-effect of causing static initialization of Metrics, starting instrumentation reporting.
+            new RestartGauge(Metrics.getRegistry(), RollupService.class);
```

Comment:
Yikes! RestartGauge() is our class. Should we have built in some logic to handle this inside the class?